examples/brainfuck/src/main.rs

```diff
@@ -172,11 +173,13 @@ fn codegen<'a, 'ctx>(ops: &[Op], context: &'a gccjit::Context<'ctx>) -> bool {
             }
             Op::Input => {
                 let access = context.new_array_access(None, array.to_rvalue(), memory_ptr.to_rvalue());
+               
```

Comment:
Please remove this new line and the others below.

Reply:
Alright